.docker/.env

```diff
@@ -0,0 +1,4 @@
+﻿MS_SQL_DB_PORT=1433
+SA_PASSWORD=Sashacool1
```

Comment:
Let's consider some random password

Reply:
We have set the default password for "sa" user, so that all team members d onot need to reset their user login every single time.

Reply:
We have changed this hardcoded password to random generated string - "JPv9bcqURjfNqzJw".